CONTRIBUTING.md

```diff
@@ -59,6 +59,21 @@ disable cross compilation.
 
 Cross compilation is set up in docker/debian.Dockerfile.
 
+## Configuration file for tests
+
+Currently, the behavior for the following test options can be configured using the TOML format:
+
+- `rustc_channel`: Specifies which Rust compiler channel to use when running tests that build Rust code.
+
+- `use_qemu`: Determines whether to run tests for architectures different from the host. This setting is overridden by the `$WILD_TEST_CROSS` environment variable.
```

Comment:
We may eventually want to be able to specify which cross-compilation toolchains are available. e.g. `cross_toolchains = ["aarch64], "riscv64"]`, but this is fine for now.